locust/USAGE.md

```diff
@@ -0,0 +1,56 @@
+## Purpose 
+This directory contains a Dockerfile that builds the [OpenSSL v3](https://github.com/openssl/openssl)  [OQS provider](https://github.com/open-quantum-safe/oqs-provider), and Python3 which allows locust to negotiate quantum-safe keys in TLS 1.3.
+
+## Start
+1) Run `docker build -t oqs-locust:0.0.1 .` to create a post quantum-enabled Locust docker image.
+2) To verify all components perform quantum-safe operations, first start the container with docker compose, setting all environment variables as needed. For example: 
+```
+LOGGER_LEVEL=DEBUG HOST=https://YOUR_QS_HOST:4433 docker compose  up --scale master=1 --scale worker=8
+```
+3) Connect to the locust web interface at `http://localhost:8189` and start a load test.
+
+By default, Locust supports all algorithms supported by the OQS openssl.
+
+Some environments variables you need to know
+- LOGGER_LEVEL: Set the log level for the locust master and worker. Default is ERROR.
+- HOST: Set the host to test. Default is https://test:4433
+- WORKERS: Set the number of workers. Default is 8. Ideally, the number of workers should be the same as the number of cores in the machine.
+- MASTER: Set the number of master. Default is 1.
```

Comment:
What is the number of master? Any number possible? What's the rationale for that? Sorry for the possibly pointless question from a person not knowing `locust` -- but then others may have the same question. So a pointer to the documentation of thus number may be helpful.

Reply:
Please feel free to ask any questions; I'm happy to provide answers. You’re absolutely right—this could indeed lead to errors. Locust, like any other master-worker cluster setup, allows only one master and multiple workers (in a previous project, I managed to run 800 workers across 200 VMs with obviously one unique master). I exposed the master variable unnecessarily, so I’ll modify it so that with docker-compose, we only need to specify the number of workers

Reply:
Done [here](https://github.com/open-quantum-safe/oqs-demos/pull/304/commits/ab1d2f45c378d14f7b5c01a1e6a3b656a69b8b3c#diff-8a1eed52bbc906e4f897ac6f113321abdd9f6fa9c742de72ac721a6d0cb6f195R4)